dask/tests/test_utils.py

```diff
@@ -683,6 +683,8 @@ def test_parse_timedelta():
         parse_timedelta("1", default=False)
     with pytest.raises(TypeError):
         parse_timedelta("1", default=None)
+    with pytest.raises(ValueError):
```

Comment:
```suggestion
    with pytest.raises(ValueError, match=r"Invalid time unit: foo. Valid units are: ..."):
```

Reply:
None of the other tests do that but sure, we can test the message.

I'm against testing the whole message, I think the specific list of units isn't part of what we're trying to test, so I put up a commit that just tests the beginning of message. Okay with you?